app/src/main/java/com/example/mycalculator/utils/StringUtils.java

```diff
@@ -7,5 +7,7 @@
     public static final String OPERATOR_MINUS = "-";
     public static final String OPERATOR_DIVIDE = "/";
     public static final String OPERATOR_MULTIPLY = "X";
+    public static final String ERROR_MESSAGE_DIVISION = "Error, can not divide by zero";
+    public static final String ERROR_MESSAGE_INVALID_FORMAT = "Error: Invalid format";
```

Comment:
do you use these strings to show them on the UI? if so, you need to add it on the String res, not to a constant. Check the error handling

Reply:
matias, you could use and enum class, with error codes, and then map the errors..